src/System.Private.CoreLib/shared/Internal/Resources/PRIExceptionInfo.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Internal.Resources
```

Comment:
@MichalStrehovsky There are a few references to this in TFS that will need to be adjusted from System.Resources namespace to Internal.Resources namespace.